### PR TITLE
UNIFY-474-sidebar-nav

### PR DESCRIPTION
**Pull request by jaffrepaul**
https://cypress-io.atlassian.net/browse/UNIFY-474

### CODE REVIEW

File: components/sidebar/CollapsibleSidebarSection.vue [hunk 1]
```diff
@@ -161,8 +161,9 @@ export default {
           </nuxt-link>
         </li>
         <li v-else>
-          <div class="pl-4 pr-2 py-0 text-md font-bold bg-gray-200">
-            {{ child.title }}
+          <div class="pl-4 py-0 mt-4 text-xs font-semibold text-gray-400 grid grid-cols-1 divide-y divide-gray-300 divide-dashed">
+            <span>{{ child.title }}</span>
+            <span></span>
```
Comment (cowboy) [inline components/sidebar/CollapsibleSidebarSection.vue:166]: Why the extra `span`?

**Review by cowboy — CHANGES_REQUESTED** [review]

File: components/sidebar/CollapsibleSidebarSection.vue (hunk 1 above)
Comment (jaffrepaul) [reply components/sidebar/CollapsibleSidebarSection.vue:166]: Yeah, experimenting with the best way to handle the dashed horizontal border across the full sidebar width, not just the content. Tailwind has [this approach](https://tailwindcss.com/docs/divide-width#add-borders-between-stacked-children) with adds the separator between children, which there is only really one of. It's not ideal but the thought was this is likely a placeholder for a convo starter with design and can be updated.

Comment (cowboy) [reply components/sidebar/CollapsibleSidebarSection.vue:166]: You should be able to put the border on the span with something like `border-b border-gray-300 border-dashed`

Comment (jaffrepaul) [reply components/sidebar/CollapsibleSidebarSection.vue:166]: Cool, that worked. Tried something similar yesterday but I think I was tacking on unnecessary integers as px value which made for wonk. There are still mobile considerations that need to be added but I'll see where the design convo goes before getting into that detail.

**Review by cowboy — APPROVED** [review]
This looks so much better! :shipit:

**Pull request merged**